### ISSUE TITLE
1.3.3 Sensory Characteristics += customisation ?

### ISSUE THREAD

**Issue by LaZay** [w3c/wcag#3155]
It might be interesting to complement the definition of the [1.3.3 Sensory Characteristics ](https://www.w3.org/TR/WCAG22/#sensory-characteristics) criteria with a requirement on sensory customisation.
So far this criteria only addresses the needs of blinds, and deaf users. It does not address the needs of low vision, hearing impaired, and DYS people.
These users need to be able to change user preferences, to adapt their reading experience to their sensory impairment. They need to choose more convenient fonts, color and back-groundcolor, size, space, sound volume values.
These users' preferences could be considered as sensory customisation. As such, it could maybe be part of the sensory characteristics. What do you think?

**Comment by LaZay** [comment]
It is because you cannot perceive these properties, and also because you can change them dynamically (in some reading environments), that "Instructions provided for understanding and operating content" should cannot only rely on them.

But, your right, my suggestion goes much too far away from the 1.3.3 criteria.
I do not how to keep the idea in mind for better  user experience on browsers though.

**Comment by LaZay** [comment]
[1.4.8 Visual Presentation](https://www.w3.org/TR/WCAG22/#visual-presentation) criterion already covers the need for visual customization. I have added a request for font selection into it (report to [issue #3660](https://github.com/w3c/wcag/issues/3660))
